single-work.php

```diff
@@ -0,0 +1,74 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+
+<?php
+if (have_posts()) {
+  while (have_posts()) {
+    the_post();
+
+    $work_cats = wp_get_post_terms($post->ID, 'work_cat');
+    $work_artists = get_post_artists($post->ID);
+    $work_details = get_post_meta($post->ID, '_igv_work_details', true);
+?>
+
+      <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+        <div class="container">
+          <div class="grid-row margin-bottom-basic">
+<?php
+      if (!empty($work_cats)) {
+?>
+            <div class="grid-item item-s-12 font-sans margin-bottom-basic"><?php echo $work_cats[0]->name; ?></div>
+<?php
+      }
+?>
+            <div class="grid-item item-s-10 item-l-6">
+              <h1 class="font-serif font-italic"><?php echo get_the_title($post->ID); ?></h1>
+
+<?php
+      if (!empty($work_artists)) {
```

Comment:
Possibly the `bws-mx-design-work.png` file in final design folder shouldn't be there? This layout is the same as the pdf but doesn't match the png

Reply:
the last 5 pages of that pdf is now blank for me. did you re-save it or something?

Reply:
but as i recall the work template wasn't included in the pdf

Reply:
yea the pdf is corrupted now

Reply:
i replaced it

Reply:
i think you were thinking the single-event template in the pdf was single-work. theyre similar